### PR TITLE
Fix: Rules edge cases

### PR DESCRIPTION
**Pull request by Meldiron**
## What does this PR do?

Fix missing rules (redirect)

remove double protocol

improve placeholders for dns records

fix www subdomain rules

## Test Plan

Manual QA

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/(console)/project-[region]-[project]/functions/function-[function]/domains/add-domain/+page.svelte [hunk 1]
```diff
@@ -72,7 +72,7 @@
             } else if (behaviour === 'REDIRECT') {
                 rule = await sdk
                     .forProject(page.params.region, page.params.project)
-                    .proxy.createRedirectRule(domainName, $regionalProtocol + redirect, statusCode);
+                    .proxy.createRedirectRule(domainName, redirect, statusCode);
```
Comment (ItzNotABug) [inline src/routes/(console)/project-[region]-[project]/functions/function-[function]/domains/add-domain/+page.svelte:75]: why is this removed? do we not need the protocol anymore? what changed that its not required anymore?

File: src/routes/(console)/project-[region]-[project]/sites/site-[site]/domains/add-domain/+page.svelte [hunk 2]
```diff
@@ -76,7 +76,7 @@
             } else if (behaviour === 'REDIRECT') {
                 rule = await sdk
                     .forProject(page.params.region, page.params.project)
-                    .proxy.createRedirectRule(domainName, $regionalProtocol + redirect, statusCode);
+                    .proxy.createRedirectRule(domainName, redirect, statusCode);
```
Comment (ItzNotABug) [inline src/routes/(console)/project-[region]-[project]/sites/site-[site]/domains/add-domain/+page.svelte:79]: same as above.

File: src/routes/(console)/project-[region]-[project]/functions/function-[function]/domains/add-domain/+page.svelte (hunk 1 above)
Comment (Meldiron) [reply src/routes/(console)/project-[region]-[project]/functions/function-[function]/domains/add-domain/+page.svelte:75]: User is expected to provude URL including hostname, port, path, query params.. And protocol too
